Formula/go@1.16.rb

```diff
@@ -17,7 +17,8 @@ class GoAT116 < Formula
 
   keg_only :versioned_formula
 
-  deprecate! date: "2022-03-15", because: :unsupported
+  # Commented out while this formula still has dependents.
+  # deprecate! date: "2022-03-15", because: :unsupported
```

Comment:
Same here. I think all dependents are deprecated, so I think it is safe to keep deprecated.

Reply:
`brew` seems to think that `kubernetes-cli@1.22` is not deprecated for some reason:
```
❯ brew ruby -e 'puts "kubernetes-cli@1.22".f.deprecated?'
false
❯ brew ruby -e 'puts "sdl".f.deprecated?'
true
```

Reply:
Ah... looks like it is planned for deprecation next month. Should be fine if we want to un-deprecate `go@1.16` and re-deprecate on 2022-08-28

Reply:
Oh, I see. It's the deprecation date. Let's just match.